autoload/lsp/handlers.vim

```diff
@@ -195,13 +191,13 @@ export def ProcessRequest(lspserver: dict<any>, request: dict<any>)
       'window/workDoneProgress/create': ProcessWorkDoneProgressCreate,
       'client/registerCapability': ProcessClientRegisterCap,
       'client/unregisterCapability': ProcessClientUnregisterCap,
-      'workspace/configuration': ProcessWorkspaceConfiguration,
-      'workspace/codeLens/refresh': ProcessUnsupportedReq,
-      'workspace/semanticTokens/refresh': ProcessUnsupportedReq
```

Comment:
i think you'd better keep this two request msg and this function, 
i guess yegappan kept it for backlog, perhaps they may impl one day.

Reply:
The code behaves the same, as it fall-though to the error message just below. If they are keept like that, then you wouldn't be able to implement your own handler.

Reply:
I don't think that I made myself clear, the code will behave exactly the same as if a handler isn't recognized the exact same error message, as the one printed in `ProcessUnsupportedReq` will be printed in line 202: https://github.com/yegappan/lsp/pull/310/files#diff-4a80aa048eaef05d2305e7a083faf0cf87c4b69a63e11faa37209fb3bb33ea72L206

Reply:
those two should be a remark, to indicate those two had not impl yet, clear?
at least i understood like that, or you can help impl those two request msg to common funcs now, then everything ok,, then you can have your customized funcs to those specific lang had specific msg.